CONTRIBUTING.md

```diff
@@ -57,6 +57,19 @@ This repo is a “monorepo,” meaning it contains several projects in one. It c
    ```sh
    pnpm i
    ```
+#### Contributing using Gitpod
```

Comment:
I’m guessing this a `<h4>` because you consider it to be a subheading of “Setting up a development environment”? In that case, maybe it would be good to add an equivalent `#### Developing locally` heading before the local development steps?

I’ll leave it up to you if this heading should stay unchanged or match whatever structure we have above, e.g. “Developing locally” vs “Developing using Gitpod” or something.

Reply:
Sure thing! Tell me if you hate this.